components/src/components/elements/included-features/IncludedFeatures.tsx

```diff
@@ -52,6 +55,9 @@ function resolveDesignProps(props: RecursivePartial<DesignProps>): DesignProps {
       isVisible: props.entitlement?.isVisible ?? true,
       fontStyle: props.entitlement?.fontStyle ?? "text",
     },
+    entitlementExpiration: {
+      fontStyle: props.entitlementExpiration?.fontStyle ?? "heading6",
```

Comment:
If we are allowing the customer/user to set a font style in the component builder, we will need to update `schematic-app` to pass this prop from the sidebar settings for the `IncludedFeatures` component. Otherwise, this will always default to `heading6`.

See [here](https://github.com/SchematicHQ/schematic-app/blob/main/frontend/src/modules/components/components/builder/elements/IncludedFeatures/index.tsx#L230-L274) for an example on how to do this.

Reply:
Okay, thank you for pointing out :) 

I will add them in separate PR then.